ERROR    root:oneup_health.py:126 1UpHealth service unavailable - Path: /user-management/v1/user - Status: 503